Dfe.Data.SearchPrototype/Web/Views/Shared/_Layout.cshtml

```diff
@@ -27,15 +27,23 @@
                 </a>
             </div>
             <div class="govuk-header__content">
-                <a class="govuk-header__link govuk-header__service-name" asp-controller="Home" asp-action="Index">Search prototype</a>
+                <a class="govuk-header__link govuk-header__service-name" asp-controller="Home" asp-action="Index">
+                    Search prototype
```

Comment:
We've lost the changes we made to the content of the anchor tags here, but we'll fix them after merge